Button/button.css

```diff
@@ -0,0 +1,244 @@
+:root{
+    --white-color:#fff;
+    --btn-hover:#333;
```

Comment:
Can we keep this property name as such? `--color-btn-hover`

Reply:
This is implemented/resolved.

Reply:
Done 👍